src/styles/utils/_mixins.scss

```diff
@@ -1,6 +1,42 @@
-@mixin hover($_property, $_toValue) {
-  transition: #{$_property} 0.3s;
+@mixin limitContentWidth {
+  margin: 0 auto;
+  max-width: 1260px;
+}
+
+@mixin tabletBreakpoint {
+  @media (min-width: $tablet-breakpoint) {
+    @content;
+  }
+}
+
+@mixin desktopBreakpoint {
+  @media (min-width: $desktop-breakpoint) {
+    @content;
+  }
+}
+
+@mixin fontParams($font-size, $font-weight, $letter-spacing: normal) {
+  font-size: $font-size;
+  font-weight: $font-weight;
+  letter-spacing: $letter-spacing;
+}
+
+@mixin marginBetweenSections($mobile, $tablet, $desktop) {
+  margin-top: $mobile;
+
+  @include tabletBreakpoint {
+    margin-top: $tablet;
+  }
+
+  @include desktopBreakpoint {
+    margin-top: $desktop;
+  }
+}
+
+@mixin scaleOnHover {
+  transition: transform 0.5s ease-in-out;
+
   &:hover {
-    #{$_property}: $_toValue;
+    transform: scale(1.5);
```

Comment:
1.5 is way too much - reduce to reasonable. Usually 1.2 is fine.

Reply:
Sorry, set it like this to highlight the issue 🤣